ppss.yaml

```diff
@@ -61,6 +61,23 @@ sim_ss: # sim only
       createMarketBuyOrderRequiresPrice": False
       defaultType: spot
 
+multisim_ss:
+  approach: simplesweep # simplesweep | fastsweep (future) | ..
```

Comment:
are these options documented somewhere?

Reply:
About as much as the other options are documented. Which means "lightly". Specifically, in:
- ppss.yaml like you see
- and CAND_APPROACHES at the top of pdr_backend/sim/multisim_engine.py

And that's ok for now. I want to keep it light so that we can evolve rapidly. "Make $" is the goal.

BTW this PR is still in draft form. Things will keep evolving.